must-gather/collection-scripts/gather_ceph_resources

```diff
@@ -396,27 +396,63 @@ for ns in $namespaces; do
         oc rsync -n "${ns}" "$(oc get pods -n "${ns}" -l "${node//./}"-debug='ready' --no-headers | awk '{print $1}')":/host/var/lib/rook/openshift-storage/crash/ "${CRASH_OUTPUT_DIR}"
     }
 
+    journal_collection(){
+        printf "collecting journal logs from node %s \n" "${node}" | tee -a  "${BASE_COLLECTION_PATH}"/gather-debug.log
+        oc rsync -n "${ns}" "$(oc get pods -n "${ns}" -l "${node//./}"-debug='ready' --no-headers | awk '{print $1}')":/host/var/log/journal.gz "${JOURNAL_OUTPUT_DIR}"
+        oc rsync -n "${ns}" "$(oc get pods -n "${ns}" -l "${node//./}"-debug='ready' --no-headers | awk '{print $1}')":/host/var/log/kernel.gz "${KERNEL_OUTPUT_DIR}"
+    }
+
+    delete_journal_log(){
+        for node in ${nodes}; do
+        printf "Deleting journal logs from node %s \n"  "${node}"
+        oc debug nodes/"${node}" << CMDS
+            chroot /host
+            cd var/log
+            rm -rf journal.gz
+            rm -rf kernel.gz
+            exit
+CMDS
+    done
+    }
     # creating a counter variable for collecting PID in array
-    pids=()
+    pids_log=()
+    
+    for node in ${nodes}; do
+        printf "Generating journal logs from node %s \n"  "${node}"
+        oc debug nodes/"${node}" << CMDS
+            chroot /host
+            cd var/log
+            journalctl --since "2 days ago" | gzip >> journal.gz
+            journalctl -k -o short-iso-precise --utc --no-hostname | gzip >> kernel.gz
```

Comment:
What is the use case for appending to a (potentially existing -- what about `delete_journal_log`?) archive instead of always generating a new one?

Reply:
keeping the log files to append next time may take unnecessary memory. Hence, from my point of view, deleting and recollecting logs every time we run the must-gather is no harm.

Reply:
But what you are doing here is appending (`>>`) rather than always generating a new archive (`>`).